src/components/Providers/ProviderListItem/ProviderListItem.js

```diff
@@ -0,0 +1,30 @@
+import { Link } from 'react-router-dom';
+import ProviderImage from '../ProviderImage/ProviderImage';
+import Label from '../../UI/Label/Label';
+import { formatProviderHeading } from '../../../helpers/utils';
+import './ProviderListItem.scss';
+
+const ProviderListItem = ({ provider }) => {
+  const { id, avatarUrl, name, title, designation, bio, availability } =
+    provider;
+
+  const providerHeading = formatProviderHeading(name, title);
+
+  return (
+    <li className="providers__item">
+      <Link to={`/providers/${id}`}>
+        <div className="providers__item--profile">
+          <ProviderImage avatarUrl={avatarUrl} size="small" />
+          <div className="providers__item--header">
+            <h2 className="providers__item--heading">{providerHeading}</h2>
+            <p className="providers__item--designation">{designation}</p>
+          </div>
+        </div>
+        <div className="providers__item--bio">{bio}</div>
+        <Label variant="availability">Available {availability}</Label>
```

Comment:
This availability needs to get transformed in the case of 'next week' to remove the dash.

![next week](https://user-images.githubusercontent.com/103941625/178288878-5787a4e1-bcd6-4bbf-a224-bd3b8d4ab57f.png)